### ISSUE TITLE
How to pre-train APNs ?

### ISSUE THREAD

**Issue by abhimahesh** [ouceduxzk/Fine_Grained_Classification#6]
I trained this model on my dataset but i am not getting much improvement in accuracy than directly using fine-tuned **googLenet**. Maybe because i didn't use pre-trained APNs. So how do i train APN?

I have following questions regarding pretraining APN :
**1.** In RA-CNN paper what does it mean to select the square from original image with the **highest response value** in the last conv5_4 layer? (conv5_4 blobs.data.shape is 32, 512, 14, 14) 
**2.** Which dataset you are using to train these APNs?

**Comment by ouceduxzk** [comment]
1. get the biggest value in the 14x14 region out the 512 channel, and you know the pos x, y
2. i use stanford car dataset
